decidim-proposals/app/packs/stylesheets/decidim/proposals/proposals.scss

```diff
@@ -103,11 +103,211 @@
+              @apply leading-6 font-semibold #{!important};
+            }
+          }
+
+          &__units {
+            @apply text-sm leading-[22px] mx-1;
+          }
+        }
+
+        .proposal_votes_container {
+          @apply flex-1 md:w-full md:flex-none p-2 lg:p-0 lg:m-3 lg:w-[164px];
+
+          button {
+            @apply text-sm leading-[18px] font-semibold py-1;
+          }
+        }
+      }
+
+      .votes_hidden {
+        @apply md:flex-none;
+
+        .proposal_votes_container {
+          @apply md:flex-none md:w-[130px] lg:w-[164px] lg:p-0;
+        }
+      }
+    }
+  }
+
+  &__grid-grid {
+    @apply md:gap-4;
+
+    .card__proposals-item {
+      @apply flex flex-col justify-between border-solid border-[#D3D5D9] border rounded;
+
+      .card__grid {
+        @apply mb-2.5 md:mb-0 flex-1 justify-between;
+
+        box-shadow: none !important;
+
+        &-text {
+          @apply justify-end lg:gap-y-8;
+        }
+
+        &-metadata {
+          @apply mt-0 md:flex md:flex-row;
+
+          > :first-child {
+            flex: 1 !important;
+          }
+
+          > :not(:first-child) {
+            @apply border-r border-[#D3D5D9];
+          }
+
+          & > div {
+            @apply px-2 md:px-2;
+          }
+        }
+      }
+
+      .card__proposals-votes {
+        @apply flex bg-[#F5F5F5EE] justify-around items-center px-4 h-[68px];
+
+        button {
+          @apply bg-[var(--secondary)];
+
+          &[disabled] {
+            @apply opacity-50 text-white border-current cursor-not-allowed;
+          }
+        }
+
+        .with_votes_limit {
+          @apply flex flex-col lg:items-start flex-1 md:mr-4 lg:mr-0;
+
+          .proposals__aside-progress {
+            @apply flex flex-col-reverse w-[80%] items-center;
+          }
+
+          .progress-bar {
+            &__number {
+              @apply lg:w-[80%];
+
+              span {
+                @apply text-sm;
+              }
+            }
+
+            &__units {
+              @apply flex justify-center w-[80%];
+            }
+          }
+        }
+
+        .without_votes_limit {
+          @apply flex flex-row flex-1 lg:flex-1 md:flex-none md:mr-4 lg:mr-0;
+
+          .progress-bar__number {
+            span {
+              @apply text-xl;
+            }
+          }
+        }
+
+        .progress-bar {
+          &__number {
+            @apply flex justify-center items-center;
+
+            span {
+              @apply leading-6 font-semibold #{!important};
+            }
+          }
+
+          &__units {
+            @apply text-sm leading-[22px] mx-1;
+          }
+        }
+
+        .proposal_votes_container {
+          @apply flex-1 lg:w-[164px] lg:flex-none;
+
+          button {
+            @apply text-sm leading-[18px] font-semibold py-1;
+
+            .already-voted-icon {
+              @apply w-4 h-4;
+            }
+          }
+        }
+      }
+
+      .votes_hidden {
```

Comment:
```suggestion
      &-votes_hidden {
```

Reply:
implemented as .card__proposals-votes-hidden

Reply:
done in[dd839af](https://github.com/decidim/decidim/pull/13883/commits/dd839af40bf2d480679578a5583a79078ff9ad0f)